src/initializers/diamorphosis.ts

```diff
@@ -23,5 +30,4 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
     ...config.riviere
   };
   diamorphosis(orkaOptions.diamorphosis);
-  config.honeybadgerEnvironment = config.honeybadgerEnvironment || config.nodeEnv;
```

Comment:
Can you add this here again? And possibly add a test for it.
The problem is that if you don't set honeybadgerEnvironment but only set NODE_EVN in env the honeybadgerEnvironment remains 'development'